set-kubectl/action.yaml

```diff
@@ -0,0 +1,10 @@
+name: Install kubectl
+description: Sets up the kubectl
+
+runs:
+  using: composite
+  steps:
+    - name: Install kubectl
+      uses: azure/setup-kubectl@v4
+      with:
+        version: 'v1.26.10'
```

Comment:
Can we change it to our current kubernetes version (1.27.7?)

Reply:
Fixed this